test/integ/test_key_pair_authentication.py

```diff
@@ -64,8 +65,12 @@ def test_get_token_from_private_key(input_account, expected_account):
     )
 
 
-@pytest.mark.skipolddriver
-def test_regionless_url_JWT_token_validity(request, db_parameters):
+@pytest.mark.internal
+@pytest.mark.skipif(
+    not RUNNING_AGAINST_LOCAL_SNOWFLAKE,
+    reason="connection timeouts occur when attempting to connect to this external account with automated testing",
+)
```

Comment:
I tried running this against Jenkins but Jenkins did not seem to like the credentials either- I was hoping this would just be run locally. I don't think it can work with either GithubActions or Jenkins.